JunitHomework/src/main/java/string/utility/NumbersModifierNotOk.java

```diff
@@ -0,0 +1,34 @@
+package string.utility;
+
+public class NumbersModifierNotOk {
+    public String before;
```

Comment:
what does this before mean? This string name is not ok. We do not create class based on how out test class will look like. 
This variable name do not have a meaningful naming.